app/helpers/money_helper.rb

```diff
@@ -1,11 +1,11 @@
 module MoneyHelper
   include MoneyRails::ActionViewExtension
 
-  def money_sign(money)
-    money > 0 ? 'success' : 'danger'
-  end
-
   def money_with_symbol(money)
     humanized_money_with_symbol(money, symbol_after_without_space: true)
   end
+
+  def bg(amount)
```

Comment:
@Gataniel please name this method more clear, like `amount_bg_class` or something similar.